docs/usage/opentelemetry.md

```diff
@@ -0,0 +1,16 @@
+---
+title: OpenTelemetry
+description: How to use OpenTelemetry with Renovate
+---
+
+# OpenTelemetry and Renovate
+
+Renovate supports OpenTelemetry which is an emerging monitoring standard.
+
+To activate the instrumentation, the environment variable `OTEL_EXPORTER_OTLP_ENDPOINT` has to be set.
+This sets the endpoint where to send the telemetry data. If this endpoint is set, all other environment variables defined
+by the [OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md)
+are supported.
```

Comment:
Not a maintainer, but I find this feature very interesting as well and looking forward to try it out. 

Especially since I'm not very versed with OpenTelemetry you can maybe add a few more information about what is traced and how to "receive" these traces. Looking at the OpenTelemetry specification, it seems that OpenTelemetry supports traces, metrics as well as logs. Does it send all three or (and I assume that) only traces? 

Is it maybe worthwhile to add a minimal MvC/demo on how one can leverage the OpenTelemetry output? Or do I just spin up an e.g. [Jaeger](https://www.jaegertracing.io/docs/1.21/getting-started) and let it push the data there?

Reply:
I will add some more documentation.   

It's true that OpenTelemetry can transfer traces, metrics and logs, but this MVC only exports traces.    
Logs specification has finished its draft phase and are now experimental. There are no finished SDKs for it yet tough.    
Metrics are stabilized right know and the JavaScript SDKs are right now in version [0.28.0](https://github.com/open-telemetry/opentelemetry-js/releases/tag/v0.28.0).      
So this MVC add only tracing, but later on there are definitively use cases for metrics such as how often are specific managers used. 

I thought this is clear after a short Google search. I will add an example, but all bigger monitoring and cloud providers allow to natively ingest OTLP already. There are also OSS which directly ingest it already ( Grafana Tempo ) or plan to support it or have experimental builds for it ( [Jaeger](https://www.jaegertracing.io/docs/1.21/opentelemetry/) )     
If direct ingestion is not supported the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-js-contrib) is used as middleware. Here you can find the full list of [supported exporters](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter)

Reply:
That comment cleared up a lot of things and I think adding the gist of that to the docs could be very helpful. 👍 

I got it working using the OpenTelemetry Collector as middleware and Zipkin as target. 
I think it is worth to explicitly mention that only `OTLP` is supported as target/protocol and other things from the SDK-spec like the [Jaeger exporter](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#jaeger-exporter) are not supported.

Reply:
Good to hear!     
I would be interested, if the currently exported data is helpful as a starter or something glaring is missing.

Reply:
Looks great so far. I think the most important metrics are included. 

What I just experimented (and thanks for the link to the OpenTelemetry Collector project), is the [spanmetricsprocessor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanmetricsprocessor), which allows to generate metrics from the traces e.g. to show how many API requests are done to which host.

Reply:
Yes, we use this processor too. It comes in handy multiple times so far

Reply:
It would be great if these examples could be documented too, now or later

Reply:
I haved added the spanmetricsprocessor to the example and docs